### ISSUE TITLE
How to insert attachments ?

### ISSUE THREAD

**Issue by perarg** [akhoury/nodebb-plugin-import-vbulletin#7]
Hello @akhoury,
what should i do in order to have my attachments to be displayed properly in posts ? My attachments are stored locally on hard disk and not saved in database. What folder should i copy and where in NodeBB folder ?

**Comment by akhoury** [comment]
you already imported using this plugin?

**Comment by perarg** [comment]
i am trying to figure out the attachment file structure... I think i am close to... A user with userid of 347 has attachments in folder ./attachments/3/4/7 and so on..

**Comment by akhoury** [comment]
that wont work out of the box, all the user-ids and the post-ids are not going to be the same.
we need to modify this plugin to pass each user's profile picture, if the attachement was in fact its profile picture.
What about topic and posts attachments? 

would you be able to provide with some sample data I can look at? also, which version of VB are u using?

**Comment by perarg** [comment]
I am writing an exporter for VB5. I am trying to understand the structure and the db schema. They have kept the filedata table the same so i think i can based on your exporter of vb4 in order to have the same functionality changing of course the queries so to bring the right values to variables.

**Comment by akhoury** [comment]
no it's not the same, they didn't delete any tables but they added a few, the most important one I saw was the `text` table, which they migrated ALL content to it, like topics, posts and few other stuff, which then you'd need to use to JOIN with `contenttype` - it's not too hard- if you get stuck, i might be able to help  much more next weekend.

**Comment by perarg** [comment]
Yes yes, i have already done all these. I have successfully imported users, topics, categories and posts. I only have to go with attachments. That's why i asked here about the vb4 structure but i think i may have figured out. Please close the issue. If you have time maybe next week we can chat at skype further more.

**Comment by MaxWaldorf** [comment]
Sorry to dig out the thread but I am facing the same issue with attachments being stored in the database...

Size is more than 2.6G and the exporter crashes constantly when it tries to read the blob...
Not sure how to workaround this issue.

I can live without attachments but would be better to have them in...


Cheers

**Comment by akhoury** [comment]
@MaxWaldorf sorry for the late reply on this, missed many emails regarding this plugin, issue with my filters.

Are you still having issues with this? or i assume you moved on.

**Comment by MaxWaldorf** [comment]
Hi!

Yes I had the issue for attachments but not avatars... (I ended up migrating but without attachments)
I will probably have a new forum to migrate soon and the issue will occur again for sure.

Cheers
